frontend/src/routes/moods/+page.server.js

```diff
@@ -0,0 +1,27 @@
+import { redirect } from '@sveltejs/kit';
+import axios from 'axios';
+
+/**
+ * Fetches data from the habits microservice via the API gateway to retrieve all habits
+ * 
+ * @param {*} serverLoadEvent 
+ * @returns
+ */
+export const load = async ({ serverLoadEvent, cookies }) => {
+  try {
+    const jwt = cookies.get('jwt');
+
+    const response = await axios.get('http://localhost:3011/moods', {
```

Comment:
The moods page to my knowledge does not retrieve any data from the microservice. If you call this request, according to our documentation on the JSON responses you'll retrieve all moods. Since the data is not used, you should consider sending request for validation to the `localhost:3011/verify` 

Refer to login page for example - load func